geode-client-protocol/src/main/java/org/apache/geode/internal/protocol/serialization/codec/JsonPdxConverter.java

```diff
@@ -17,19 +17,29 @@
 import org.apache.geode.annotations.Experimental;
```

Comment:
Hmm, curiously I can't seem to add a comment to line 15, but did we want to rethink the package name now that we're no longer calling these codecs?